docs/sdk/src/guides/parachains/async_backing_guide.rs

```diff
@@ -0,0 +1,452 @@
+//! # Upgrade Parachain for Asynchronous Backing Compatibility
+//!
+//! This guide is relevant for cumulus based parachain projects started in 2023 or before. Later
+//! projects should already be async backing compatible. If starting a new parachain project, please use
+//! an async backing compatible template such as
+//! [`cumulus/parachain-template`](https://github.com/paritytech/polkadot-sdk/tree/master/templates/parachain).
+//!
+//! The rollout process for Async Backing has three phases. Phases 1 and 2 below put new infrastructure
+//! in place. Then we can simply turn on async backing in phase 3. But first, some pre-reqs and context
+//! to set the stage.
+//! ## Async Backing Prerequisites
+//!
+//! For more contextual information about asynchronous backing, see
+//! [this page](https://wiki.polkadot.network/docs/learn-async-backing).
+//!
+//! Pull the latest version of Cumulus for use with your parachain. It contains necessary changes for
+//! async backing compatibility. Latest on master branch of
+//! [Polkadot-SDK](https://github.com/paritytech/polkadot-sdk) is currently sufficient. Any 2024 release
+//! will work as well.
+//!
+//! ## Async Backing Terminology and Parameters
+//!
+//! Time for a bit of context before we get started. The following concepts will aid in demystifying the
+//! collator side of Async Backing and establish a basic understanding of the changes being made:
+//!
+//! - **Unincluded segment** - From the perspective of a parachain block under construction, the
```

Comment:
Why do parachain developers need to know these concepts?

I would argue 99% of the possible readers of this page don't want to know this information, and rather just know: 

1. with async backing I can have faster block times
2. how shoudl I enable it. 

 I feel like this type of information is out of scope both for here and in the wiki, and should be in an in-depth writing like parachain's implementor guide.

Reply:
Would you happen to know where the GitHub repo for these docs is? https://paritytech.github.io/polkadot-sdk/book/index.html The top right link points to the Polkadot SDK repo. 

For now, I can remove this content from this guide and keep in Wiki, till we can find a home for them on Parachain Implementors guide.